source/SIL.AppBuilder.Portal.Frontend/src/ui/components/inputs/organization-select/__tests__/page.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  interactor,
+  Interactor,
+  text,
+  is,
+  collection
+} from '@bigtest/interactor';
+
+class OrgSelect {
+
+  isDisabled = is('.disabled');
+  selected = text('.text[role="alert"]');
```

Comment:
Probably selected is not a good name for this because you are actually getting a text

Reply:
why's that?

usage:
```ts
      expect(page.orgSelect.selected).to.include('All Organizations');
```

I think it reads ok

Reply:
ok that looks good when you put it in context